### PR TITLE
Issue 115 dropdown

### PR DESCRIPTION
**Pull request by khutchison**
- added dropdown menu in data exploration field to choose between grouping by census tract or zip code
- now only one multi-select field will be shown at a time

Looks good on my end, let me know if there are any issues.

Cheers,
Kirk

### CODE REVIEW

File: local.env~ [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+MAPBOX_API_KEY=
+LOGGLY_TOKEN=
+ALGOLIA_APP_ID=
+ALGOLIA_API_KEY=
+
+# MLab's Big Query Configuration #
+##################################
+MLAB_BIGQUERY_EMAIL=
+MLAB_BIGQUERY_PRIVATE_KEY=
+MLAB_BIGQUERY_PRIVATE_KEY_PASSPHRASE=
```
Comment (Kourchenko) [inline local.env~]: Don't commit this file. Looks like it wasn't ignored by `.gitignore` because it's named `local.env~`

**Comment by Kourchenko** [comment]
One thing I noticed was the dropdown has two search icons (two magnifying glass icons).

It's due to the fact that our div container are using a class that use a background image (class="chosen-container chosen-container-single"), then one of it's children element (an input) uses the same classes, which causes the background image to be used again.

The fix is to add
```
.chosen-container-single, .chosen-search {
  background: none !important;
}
```
to the `show.scss` file. 

We want to show the search icon around the input field, which is used by the class
`.chosen-container-single .chosen-search input[type="text"]`.



None of these show up inside our project, but are imported as an asset SCSS script and applied when the browser loads. We thus have to override the functionality inside our SCSS files.

File: local.env~ (hunk 1 above)
Comment (khutchison) [reply local.env~]: Weird, looks like that was auto-generated by emacs in addition to the normal local.env.

**Comment by khutchison** [comment]
hi all, @Kourchenko implemented the fix as you suggested and it worked great, thank you. \

@ryanrolds changed it so the previous filter's 'disabled' option gets toggled when the user switches. This way it won't affect the graphs but the options they selected will still be there if the user switches back to the previous filter.

**Comment by ryanrolds** [comment]
You're really close.

**Pull request merged**